dashboard/app/models/pd/application/teacher1920_application.rb

```diff
@@ -204,7 +210,7 @@ def dynamic_required_fields(hash)
         end
 
         if hash[:pay_fee] == TEXT_FIELDS[:no_pay_fee_1920]
-          required.contact [:scholarship_reasons]
```

Comment:
lol, good find.

We must be missing some test coverage here. I'd expect a test to have exposed that.